ui/src/components/Player/index.tsx

```diff
@@ -142,7 +142,7 @@ export default class Player extends React.Component<IProps> {
                             <div className="player-podcast-name">
                                 {episode.feedTitle !== undefined ?
                                     <Link to={`/podcast/${episode.feedId}`} title={episode.feedTitle}>
-                                        from: {episode.feedTitle}
+                                        {`from: ${episode.feedTitle}`}
```

Comment:
It appears that "from:" is now in the line as text. Should only show the subtitle here.

Mainly referring to the images you posted.

Reply:
Okay, sounds good. Thanks for the PR!